src/actions/motion.ts

```diff
@@ -68,16 +68,18 @@ abstract class MoveByScreenLine extends BaseMovement {
         // for some reason vscode doesn't update them. But doing it this way does
         // update vscode's selections.
         const selections = vimState.editor.selections;
```

Comment:
Can we just collapse these 3 lines to `vimState.editor.selections = vimState.editor.selections.map(...)`?

Reply:
Sure.